### PR TITLE
MAT-9503: reference update

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-9503](https://jira.cms.gov/browse/MAT-9503)
Jira Ticket: [MAT-9566](https://jira.cms.gov/browse/MAT-9566)

(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by mcmcphillips** [comment]
image

**Comment by mcmcphillips** [comment]
image

File: src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/ReferenceComponent.tsx [hunk 1]
```diff
@@ -308,6 +318,80 @@ export default function ReferenceComponent({
...
+              </MenuItem>
+            ))}
+            onChange={(e) => {
+              setSelectedProfileAddNew(e.target.value);
+            }}
+            placeHolder={{
+              name: "Select",
+              value: "",
+            }}
+            data-testId=""
```
Comment (adongare) [inline src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/ReferenceComponent.tsx]: this could be removed?

File: src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/ReferenceComponent.tsx [hunk 2]
```diff
@@ -308,6 +318,80 @@ export default function ReferenceComponent({
           <div />
         </div>
       )}
+      <MadieDialog
+        form
+        title="Chose Profile"
```
Comment (adongare) [inline src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/ReferenceComponent.tsx]: should this be `choose` or `chose`?

Comment (mcmcphillips) [reply src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/ReferenceComponent.tsx]: It should be. I copy pasted the ticket instead

**Review by adongare — APPROVED** [review]
